pkgs/development/python-modules/tensorflow/1/default.nix

```diff
@@ -255,6 +255,9 @@ let
       # include security vulnerabilities. So we make it optional.
       # https://github.com/tensorflow/tensorflow/issues/20280#issuecomment-400230560
       sed -i '/tensorboard >=/d' tensorflow/tools/pip_package/setup.py
+
+      substituteInPlace tensorflow/tools/pip_package/setup.py \
+        --replace "numpy >= 1.16.0, < 1.19.0" "numpy >= 1.16.0"
```

Comment:
there is an abi breakage with numpy 1.19. Not sure what we should do about it in nixpkgs